package.json

```diff
@@ -25,21 +26,23 @@
     "url": "https://github.com/alexanderzobnin/grafana-zabbix/issues"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-react": "^7.6.3",
-    "@grafana/data": "^6.4.3",
-    "@grafana/runtime": "^6.4.3",
-    "@grafana/toolkit": "^6.4.3",
-    "@grafana/ui": "^6.4.3",
+    "@babel/core": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
+    "@emotion/core": "^10.0.27",
+    "@grafana/data": "canary",
+    "@grafana/runtime": "canary",
+    "@grafana/toolkit": "canary",
+    "@grafana/ui": "canary",
```

Comment:
There's currently an issue that's failing unit tests if we don't mock @grafana/ui and @grafana/runtime: https://github.com/grafana/grafana/issues/20921

However there's also apparently an issue in master that's causing a different error entirely. Neither one works, but we're going to need 6.6 (or whatever release has the fix).

Basically Jest tests are broken because of Grafana packages, we can't fix it and need to wait for a good version of @grafana

Reply:
Aha, ok, that makes sense. I'll try to investigate. It seems, I remember some build error in grafana/ui related to Table interface.